### PR TITLE
Ports the portocol diagram to ASCII 

### PR DESCRIPTION
**Pull request by samuelgoto**
https://developer.chrome.com/blog/fedcm-origin-trial/

***
<a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/pull/269.html" title="Last updated on Jun 8, 2022, 12:40 AM UTC (f1f6a5f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/269/1ccd28d...f1f6a5f.html" title="Last updated on Jun 8, 2022, 12:40 AM UTC (f1f6a5f)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -108,6 +113,31 @@ third-party cookies the API places the [=User Agent=] as a mediator between
 [=RPs=] and [=IDPs=]. This mediation requires user consent before permitting
 the [=RPs=] and [=IDPs=] to know about their connection to the user.
 
+<script type="text/svgbob">
```
Comment (npm1) [inline spec/index.bs:116]: Dupe from the other PR?

File: spec/index.bs [hunk 2]
```diff
@@ -559,16 +589,108 @@ The [[#idp-api]] and the [[#rp-api]] defines a set of HTTP APIs that cooperating
...
+│           │                  │    Logout +-*  ◄────────┘        │           │
+│           │                  │           │                      │           │
+└───────────┘                  └───────────┘                      └───────────┘
+</script>
 
 The user agent intermediates in such a matter that makes it impractical for the
 API to be used for tracking purposes, while preserving the functionality of
 identity federation.
 
+<script type="text/svgbob">
```
Comment (npm1) [inline spec/index.bs:629]: Looks like this is the newer one... looks good!

File: spec/index.bs [hunk 3]
```diff
@@ -108,6 +113,31 @@ third-party cookies the API places the [=User Agent=] as a mediator between
...
+| | "https://rp.example"        | |       | | "https://rp.example"        | |
+| └─────────────────────────────┘ |       | └─────────────────────────────┘ |
+| ┌─────────────────────────────┐ |       | ┌─────────────────────────────┐ |
+| |                             | |       | |                             | |
+| |   Welcome to my website!    | |       | |  Welcome to my website!     | |
+| |                             | |       | |                             | |
+| |                             | |       | |                             | |
+| +─────────────────────────────+ |       | +─────────────────────────────+ |
+| |       Choose an account     | |       | |    Sign-in to rp.example    | |
+| |   to sign-in to rp.example  | |       | |      with idp.example?      | |
```
Comment (npm1) [inline spec/index.bs:128]: I wonder why there's so much space between words in these diagrams? They look a bit off

Comment (samuelgoto) [reply spec/index.bs:128]: Yeah, strings are handled differently, and I can use "text here" to make them render better. I'll take another pass at them on a separate CL, this is just bringing in mocks from the explainer.

File: spec/index.bs (hunk 1 above)
Comment (samuelgoto) [reply spec/index.bs]: Yes, this is branched from the branch, so will be removed once I merge the baseline.

**Review by npm1 — APPROVED** [review]
LGTM modulo rebasing. This has 268s changes. Please dont send a stack like this as it would make reviewing very hard.

**Comment by samuelgoto** [comment]
> LGTM modulo rebasing. This has 268s changes. Please dont send a stack like this as it would make reviewing very hard.

Will do, need to figure out how to work with branches in github! Thanks!

**Pull request merged**